src/TaylorBot.Net/Program.Commands.Discord/src/TaylorBot.Net.Commands.Discord.Program/Modules/DiscordInfo/Commands/DiscordInfoModule.cs

```diff
@@ -155,7 +155,7 @@ private static Color GetColorFromStatus(UserStatus userStatus)
                         await member.GetTrackedUserAsync();
 
                     var embed = new EmbedBuilder()
-                        .WithUserAsAuthor(guildUser)
+                        .WithUserAsAuthor(guildUser, false)
```

Comment:
This is minor but I usually use [named arguments](https://learn.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments) for bools as it makes reading code a bit easier (some IDEs even simulate this feature when viewing a file)
```suggestion
                        .WithUserAsAuthor(guildUser, showGuildAvatar: false)
```